3scale/porta
# Highlight active context / menu in ContextSelector

**What this PR does / why we need it**:

According to the mockup, the active menu AKA selected context has to be highlighted.
Also, when having only 1 service the ContextSelector is not hidden anymore. Only the `input` element is hidden instead.

ApiSearch and ContextSelector have fused
![](https://media.giphy.com/media/TbYgHMnICI1A4/giphy.gif)

**Verification steps** 

1. Go to Dashboard, Dashboard only should be highlighted.
2. Go to Audience, Audience only should be highlighted regardless of the submenu selected.
3. Go to any Service, that service only should be highlighed regardless of the submenu selected.
4. When having 1 service only, the dropdown should be available but no input element inside.

**Special notes for your reviewer**:

The _Overview_ page of any service with make `ContextSelector` fail for now.
